### PR TITLE
Bugfix/fix safety invariant

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radixdlt/src/integration/java/com/radixdlt/consensus/simulation/invariants/bft/SafetyInvariant.java [hunk 1]
```diff
@@ -91,48 +85,40 @@
...
-				final View lowest = network.getNodes().stream()
-					.map(n -> lastCommittedByNode.getOrDefault(n, View.genesis()))
+				lastCommittedByNode.put(node, epochView);
+				final EpochView lowest = network.getNodes().stream()
+					.map(n -> lastCommittedByNode.getOrDefault(n, EpochView.of(0, View.genesis())))
 					.reduce((v0, v1) -> v0.compareTo(v1) < 0 ? v0 : v1)
-					.orElse(View.genesis());
-				final Set<View> viewsToRemove = committedVertices.keySet().stream()
+					.orElse(EpochView.of(0, View.genesis()));
+				final Set<EpochView> viewsToRemove = committedVertices.keySet().stream()
```
Comment (clstrfsck) [inline radixdlt/src/integration/java/com/radixdlt/consensus/simulation/invariants/bft/SafetyInvariant.java]: Could replace this with `committedVertices.headMap(lowest).clear()`.

File: radixdlt/src/main/java/com/radixdlt/consensus/bft/VertexStore.java [hunk 2]
```diff
@@ -519,12 +519,12 @@ public VertexMetadata insertVertex(Vertex vertex) throws VertexInsertionExceptio
 			vertex = vertices.remove(vertex.getParentId());
 		}
 
-		for (Vertex committed : path) {
-			CommittedAtom committedAtom = new CommittedAtom(committed.getAtom(), commitMetadata);
+		for (Vertex committedVertex : path) {
```
Comment (clstrfsck) [inline radixdlt/src/main/java/com/radixdlt/consensus/bft/VertexStore.java]: This change doesn't seem to be doing anything useful.

**Review by clstrfsck — APPROVED** [review]
Looks good.  Couple of minor suggestions not affecting behaviour.

File: radixdlt/src/integration/java/com/radixdlt/consensus/simulation/invariants/bft/SafetyInvariant.java (hunk 1 above)
Comment (talekhinezh) [reply radixdlt/src/integration/java/com/radixdlt/consensus/simulation/invariants/bft/SafetyInvariant.java]: ooo nice

**Pull request closed without merging**